index.css

```diff
@@ -0,0 +1,224 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+body {
+  overflow-x: hidden;
+  /* font-family: 'Noto Sans', sans-serif;
+font-family: 'Montserrat', sans-serif; */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Montserrat', sans-serif;
+}
+/* 
+::-webkit-scrollbar {
+  width: 8px;
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: var(--primary-color);
+  border-radius: 2rem;
+} */
+
+.home-navbar {
+  background-color: white;
+  position: fixed;
+  top: 0;
+  z-index: 99;
+  padding: 8px 5rem;
+}
+
+.home-section {
+  margin-top: 2rem;
+  height: 100vh;
+  width: 100vw;
+  margin-bottom: 2rem;
+  padding: 1rem;
+}
+
+.home-cards-section {
+  display: flex;
+  margin-top: 1rem;
+}
+
+/* header section*/
+
+.hero-section {
+  background: url('/assets/image/hero-image/N29_no_background.png') no-repeat
+    center;
+  background-color: black;
+  background-size: contain;
+  padding: 2rem 5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.hero-section .btn {
+  box-shadow: none;
+}
+
+.hero-container {
+  width: 50vw;
+  min-width: 350px;
```

Comment:
it would be better if you can use rem instead of px

Reply:
Thanks, will change it.